fuel/transformers/__init__.py

```diff
@@ -22,10 +22,14 @@ class Transformer(AbstractDataStream):
         this attribute. Use it to access data from the wrapped data stream
         by calling ``next(self.child_epoch_iterator)``.
 
+    batch : boolean
```

Comment:
`batch_input` instead of `batch`, and there should be no new line between parameter descriptions.